modules/hcp-virt-add-node.adoc

```diff
@@ -81,4 +81,4 @@ example                   example         5               5               False
 example-extra-cpu         example         2               2               False         False        4.x.0
 ----
 +
-Replace `4.x.0` with the supported {product-title} version that you want to use.
\ No newline at end of file
+Replace `<4.x.0>` with the supported {product-title} version that you want to use.
```

Comment:
Similar comment here -- be sure to match the text on line 81 with the text in the annotation.